stetho/src/main/java/com/facebook/stetho/inspector/network/NetworkEventReporter.java

```diff
@@ -25,10 +25,31 @@
  *                   `-----------------------------`--------> httpExchangeFailed
  * </pre>
  *
- * Note that {@link #interpretResponseStream} combined with {@link DefaultResponseHandler}
+ * <p>Note that {@link #interpretResponseStream} combined with {@link DefaultResponseHandler}
  * will automatically invoke {@link #dataReceived}, {@link #responseReadFailed} and
  * {@link #responseReadFinished}.  If you use your own custom {@link ResponseHandler} you
- * must be sure to invoke these methods manually.
+ * must be sure to invoke these methods manually.</p>
+ *
+ * <p>For arbitrary sockets or explicitly for WebSockets, the following call flow must be met:</p>
```

Comment:
I do want to be clear that even though the API looks websocket specific that it can do any socket (by describing both I think it makes that clear)